Resources/public/js/views/serverside/ez-roleserversideview.js

```diff
@@ -82,5 +84,23 @@ YUI.add('ez-roleserversideview', function (Y) {
         _uiUnsetAssignRoleLoading: function (button) {
             button.removeClass('is-loading').set('disabled', false);
         },
+
+        /**
+         * Refreshes the role view, showing the assignments tab.
+         *
+         * @method _refreshAssignmentsTab
+         * @protected
+         * @param {Y.Node} button
+         */
+        _refreshAssignmentsTab: function (button) {
+            /**
+             * Fired when the view needs to be refreshed.
+             * @event refreshAssignmentsTab
+             * @param roleId {String} The ID of the role we are showing assignments for
+             */
+            this.fire('refreshAssignmentsTab', {
```

Comment:
@dpobel 
> maybe we need to add some code
Which is what I'm doing here, remember? ;)

> we'll be able to remove the tabId parameter
That's the big question, how? Please advise on how to reload a page and stay on the same tab, without passing a parameter. (The refactoring you suggest shouldn't be a problem.)